index.html

```diff
@@ -139,21 +139,53 @@
 
     <section>
       <h2>Introduction</h2>
+      <p>
+Cryptographic proofs enable functionality that's extremely useful to
+implementors of distributed systems. For example, proofs can be used for
+purposes such as:
+      </p>
+
+<ul>
+  <li>
+Make statements that can be shared without loss of trust, because their
+authorship can be verified by a third party (e.g. Verifiable Credentials,
+social media posts, so on).
+  </li>
+  <li>
+Authenticate as the controller of a Decentralized Identifier (DID) to some
+domain.
```

Comment:
You don't authenticate as the controller of a DID but as the DID subject. This statement would only be true conditionally: if the DID subject *happens* to be the same as the controller. It is also too specific, so I recommend:

```
Authenticate as an entity identified by a particular identifier, for example, as the subject identified by a Decentralized Identifier (DID), i.e., a DID subject.
```

Reply:
Thanks, changed.